src/google/protobuf/compiler/command_line_interface_unittest.cc

```diff
@@ -190,7 +188,9 @@ class CommandLineInterfaceTest : public testing::Test {
                                      const string& message_name);
   void CheckGeneratedAnnotations(const string& name, const string& file);
 
+#if defined(_WIN32)
   void ExpectNullCodeGeneratorCalled(const string& parameter);
```

Comment:
Since this member function is only used by two Windows-only tests, could you try moving it to a stand-alone function defined right before those Windows tests? I think that would also help simplify the ifdef logic.